script.js

```diff
@@ -0,0 +1,71 @@
+const form=document.getElementById('form');
+const username=document.getElementById('username');
+const password=document.getElementById('password');
+let isFormSubmitted = false; 
+form.addEventListener('submit', e => {
+    e.preventDefault();
+    isFormSubmitted = true;
+     validateInputs();
+});
+const isValidPassword = password =>{
```

Comment:
Also, try adding space between your **variables definition** and **functions** for more neatness! You can use a vs code extension named **prettier**. also use it to format other files **css, HTML**